Source/Serilog.Enrichers.Span/LoggerEnrichmentConfigurationExtensions.cs

```diff
@@ -15,12 +15,28 @@ public static class LoggerEnrichmentConfigurationExtensions
         /// <param name="loggerEnrichmentConfiguration">The enrichment configuration.</param>
         /// <returns>Configuration object allowing method chaining.</returns>
         public static LoggerConfiguration WithSpan(this LoggerEnrichmentConfiguration loggerEnrichmentConfiguration)
+        {
+            return loggerEnrichmentConfiguration.WithSpan(false);
+        }
+
+        /// <summary>
+        /// Enrich logger output with span information from the current <see cref="Activity"/>.
+        /// </summary>
+        /// <param name="loggerEnrichmentConfiguration">The enrichment configuration.</param>
+        /// <param name="includeTags">Adds current Activity tags to the log event.</param>
+        /// <returns>Configuration object allowing method chaining.</returns>
+        public static LoggerConfiguration WithSpan(this LoggerEnrichmentConfiguration loggerEnrichmentConfiguration, bool includeTags)
```

Comment:
For future extensibility, it would be helpful to use an options class here:

```cs
public class SpanOptions
{
    public bool IncludeTags = false;
}
```